lib/angular-tree-component.ts

```diff
@@ -66,12 +66,6 @@ import './polyfills';
   providers: []
 })
 export class TreeModule {
-  static forRoot(): ModuleWithProviders {
```

Comment:
Correct me if I'm wrong - this is a breaking change. Anyone using 'forRoot' will have their code break